chart/k8gb/values.yaml

```diff
@@ -47,7 +47,7 @@ k8gb:
 
 externaldns:
   # -- external-dns image repo:tag
```

Comment:
We should put a comment that this is important. Cause if you override an image to a private registry source could be an upstream one.

Reply:
Thanks, PR amended with extended comment.